core/src/main/resources/application-test.properties

```diff
@@ -0,0 +1,27 @@
+# Hibernate
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
```

Comment:
it's better to declare your properties in hierarchical way. For example:
instead of 

greencity.liq-pay-public-key= 2839482394892
greencity.liq-pay-private-key= 123983427788

declare:

greencity.liq-pay.public-key= 2839482394892
greencity.liq-pay.private-key= 123983427788

Which means, those 2 properties have the same root: greencity.liq-pay